codex-rs/core/src/codex.rs

```diff
@@ -165,6 +166,10 @@ impl Codex {
 
         let user_instructions = get_user_instructions(&config).await;
 
+        let exec_policy_v2 =
```

Comment:
I don't think we need to carry this `v2` around. Once we flesh out this implementation, I think we delete the original `execpolicy` crate and rename `execpolicy2` to `execpolicy`.